docs/encyclopedia/temporal-nexus-security.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: nexus-security
+title: Security in Nexus
+sidebar_label: Security
+description: Explore the security aspects of Temporal Nexus, including managing Nexus Endpoints, runtime access control, secure routing, and payload encryption to ensure safe and reliable cross-namespace operations.
+toc_max_heading_level: 4
+slug: /temporalnexus/security
+keywords:
+  - Nexus security
+  - access control
+  - encryption
+tags:
+  - security
+  - temporal
+  - distributed systems
+---
+
+### Security {#security}
+
+#### Managing Nexus Endpoints in the Nexus API Registry
+
+Like `Namespaces`, a Nexus `Endpoint` is an account-scoped resource that is global within your Temporal Cloud account.
+
+Any `Developer` in an account, who is also a `Namespace Admin` on the endpoint’s target Namespace, can manage (create/update/delete) a Nexus `Endpoint`.\
+All users with a `Read-only` role or higher in an account, can view and browse the full list of Endpoints.
+
+#### Runtime Access Control {#runtime-access-control}
+
+Access to a Nexus `Endpoint` is controlled by an allowlist policy (of caller Namespaces) for each endpoint in the Nexus API registry.
+
+Workers in each Namespace may authenticate with Temporal Cloud as they do today with mTLS certificates or the upcoming API key functionality. Once a worker has authenticated it can send Nexus `Operation` commands to Temporal Cloud using a Temporal SDK to start a Nexus `Operation` in a different Namespace. For example in the Temporal Go SDK a caller Workflow would use `nexusClient.ExecuteOperation` to issue a command to start a Nexus `Operation`.
+
+To process a `ScheduleNexusOperation` command from a caller Workflow, Temporal Cloud obtains the handler Namespace and task-queue for the handler endpoint, and restricts access by verifying that the caller’s Namespace is in the endpoint’s allowlist.
+
+In this way, Temporal Cloud acts as a trusted broker across Namespace boundaries, and relies on authenticated workers in each Namespace. There is currently no provision for a handler’s endpoint to require a specific level of caller worker authentication, so either mTLS or API keys will be allowed as defined in the caller Namespace configuration. Nexus is designed to support additional Nexus access control policies in the future.
```

Comment:
```suggestion
In this way, Temporal Cloud acts as a trusted broker across Namespace boundaries, and relies on authenticated workers in each Namespace. 
There is currently no provision for a handler’s endpoint to require a specific level of caller worker authentication, so either mTLS or API keys will be allowed as defined in the caller Namespace configuration. 
Nexus is designed to support additional Nexus access control policies in the future.
```

Reply:
I would probably just remove the "future" references and mTLS / API key mentions here and move the point about Cloud being a trusted broker to the paragraph above.

Reply:
Agreed.